Common/Data/SubscriptionManager.cs

```diff
@@ -39,7 +39,8 @@ public class SubscriptionManager
         /// <summary>
         ///     Returns an IEnumerable of Subscriptions
         /// </summary>
-        public IEnumerable<SubscriptionDataConfig> Subscriptions => _subscriptionManager.SubscriptionManagerSubscriptions;
+        /// <remarks>Will not return internal subscriptions</remarks>
+        public IEnumerable<SubscriptionDataConfig> Subscriptions => _subscriptionManager.SubscriptionManagerSubscriptions.Where(config => !config.IsInternalFeed);
```

Comment:
Any concerns here regarding internal currency conversion feeds?

Reply:
Not really, the idea is that the algorithm does not need to know or have access directly to any of these subscriptions - note that the backend/data feed stack do has direct access to the entire subscription collection.
And it's not used when we `EnsureCurrencyDataFeed()` https://github.com/QuantConnect/Lean/blob/master/Common/Securities/Cash.cs#L168